jscomp/test/label_uncurry.js

```diff
@@ -15,7 +15,7 @@ function u1(f) {
   console.log(f(2, "x"));
 }
 
-function h(unit) {
+function h(x) {
```

Comment:
The test was `x as unit` here restored the intention `x : unit` which restores the name `x`.